crates/rpc/src/v02/types.rs

```diff
@@ -866,6 +872,62 @@ pub mod reply {
         pub overall_fee: ethers::types::H256,
     }
 
+    #[derive(Debug, Deserialize, Serialize)]
+    #[serde(deny_unknown_fields)]
+    pub struct TransactionSimulation {
```

Comment:
These types are used just for the inter-process communication?

If yes, then these should be moved to under `ext_py` (the rest of the types in this file are the types for actual RPC responses).

Reply:
Yes, makes sense, moving to `ext_py`.

Reply:
Done